pkg/devspace/config/versions/latest/schema.go

```diff
@@ -582,6 +585,39 @@ type CustomConfigCommand struct {
 	OperatingSystem string `yaml:"os,omitempty" json:"os,omitempty"`
 }
 
+// LocalRegistryConfig holds the configuration of the local image registry
+type LocalRegistryConfig struct {
+	// Disable disables the local registry when true. True by default
```

Comment:
So the local registry is disabled by default?

Reply:
Oops. Will update that comment.

Reply:
converted to this schema remove ambiguity
```
localRegistry:
  enabled: *bool
```